### PR TITLE
Remove sentry elements at the start of ways/relations

### PR DESCRIPTION
**Pull request by VeaaC**

### CODE REVIEW

**Comment by gferon** [comment]
LGTM, but that gave me an additional idea - why not build a very small region in Travis CI to validate those kind of changes?

**Review by gferon — APPROVED** [review]

File: src/ids.rs [hunk 1]
```diff
@@ -121,7 +116,7 @@ mod test {
     #[test]
     fn test_large_indices() {
         let mut builder = IdTableBuilder::new();
-        builder.skip(1u64 << 33);
+        builder.next_id += 1u64 << 33;
```
Comment (boxdot) [inline src/ids.rs:119]: I have a feeling that we could also hide this bitshifts into function calls as beautification. But feel to merge if you don't want.

**Review by boxdot — APPROVED** [review]
Look really better now. Thanks.

File: src/ids.rs (hunk 1 above)
Comment (boxdot) [reply src/ids.rs:119]: I merge.

**Pull request merged**